packages/flutter/lib/src/widgets/scroll_position_with_single_context.dart

```diff
@@ -17,6 +17,21 @@ import 'scroll_notification.dart';
 import 'scroll_physics.dart';
 import 'scroll_position.dart';
 
+class JumpSimulation extends Simulation {
+  final double destination;
+
+  JumpSimulation(this.destination);
+
+  @override
+  double dx(double time) => 0.0;
+
+  @override
+  bool isDone(double time) => true;
+
+  @override
+  double x(double time) => destination;
+}
+
```

Comment:
we should be able to just jump if we don't actually want to animate. saves us the resources of creating the animation, etc.